### DISCUSSION TITLE
Assembleia Associação Python Brasil 2022

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#43] [Geral]
# Assembleia APyB 2022

Prezadas/os/es anunciamos a próxima Assembleia da Python Brasil, que será realizada no dia 23/10 às 13h no Centro de Convenções Vasco Vasques em Manaus-AM, durante a Python Brasil 2022.
    
Lembramos de que para participar da Assembleia precisa associar-se à APyB (mais informações neste site: https://python.org.br/apyb/#associar) e poderá participar das discussões na Assembleia.
    
Nessa reunião, escolheremos a cidade-sede da Python Brasil 2024, apresentaremos novas demandas da comunidade e outras demandas.
    
Contamos com a sua presença e venha contribuir com a comunidade Python brasileira!

Para mais informações, acesse: https://python.org.br/assembleia2022/

![Assembleia Geral 2022](https://user-images.githubusercontent.com/12106738/190519027-54ba5c02-e6ee-41f9-89d9-5b67f58c0ece.png)

**Comment by cecivieira** [comment]
Sugestão para pauta:

- [Fundo para Inclusão](https://python.org.br/fundo-para-inclus%C3%A3o/)
    - Apresentação dos critérios
    - Impacto na Python Brasil 2022

**Comment by luzfcb** [comment]
Sugestão de pauta:

- [Grupo do Facebook](https://www.facebook.com/groups/python.brasil), o que fazer com ele.  O grupo não é da APyB (foi criado por terceiros), contudo ele pode impactar em como Python e "comunidade" é visto no brasil e nova geração. Alguns membros da APyB são administradores dele, contudo ele não possui moderadores frequentemente ativos e proativos. Em minha experiencia pessoal é difícil e dispendioso de moderar. Muitas contas fake, offtopic, pirataria, acidez e etc.

**Reply by gabubellon** [reply]
Sobre esse ponto e add +1:
* Talvez seja o momento da APyb ter uma cadeira de "Pessoa/Gerente/qq nome de cargo" da Comunidade, ligada a diretoria de comunicação, mas focada só em mapear e entender o que fica e é e como é gerenciado pela APyB e o que não é

**Reply by rennerocha** [reply]
Faz sentido hoje em dia ter uma presença no Facebook?

**Reply by luzfcb** [reply]
Na minha opinião, o grupo poderia ser fechado e no lugar direcionar para a maillist

**Comment by tiidadavena** [comment]
Ter uma consultoria jurídica dedicada para ajudar na organização de eventos

**Reply by tiidadavena** [reply]
Tópico iniciado: vide #62

**Comment by cecivieira** [comment]
Trocar nome do prêmio.

**Reply by Bigua** [reply]
Sugiro o nome " pessoas > tecnologia ", principalmente porque do que eu sei dos dois atuais homenageados eles eram epítomes desta frase.

**Reply by rennerocha** [reply]
https://pt.wikipedia.org/wiki/Ep%C3%ADtome

**Comment by Kadidjah** [comment]
À 
Associação Python Brasil, 
APyB 
#49 
Sugiro uma pauta: análise do formato híbrido para a Asembleia Associação Python Brasil 2024 para a participação de tod@s associad@s.
Cordialmente, 
Kadidja Valéria (PyladiesDF)

**Comment by tiidadavena** [comment]
Uso da linguagem neutra nas comunicações oficiais na APyB

**Reply by tiidadavena** [reply]
Tópico puxado na seção #64

**Comment by maribedran** [comment]
Sugestão de pauta
Alteração do estatuto para que membros estudantes tenham os mesmos direitos de outros membros

**Reply by luzfcb** [reply]
Eu realmente não curto essa proposta. Isso é a única coisa que diferencia o membro regular do membro estudante, e não ajudará a fazer um membro estudante pagar a associação mais cara. Desde que manter o CNPJ da APyB tem um custo, eu não acho viável a longo prazo ficar sempre dependente de sobrar dinheiro de evento para pagar o custo fixo da APyB. Além disso, nem toda PyBR termina com caixa positivo. Se não estiver enganado, a PyBR do Rio, terminou no negativo ou pelo menos ficou no negativo até o início do evento, e se não fosse o Cadu Leite tirar mais de 10 mil do próprio bolso, o evento tinha chance de nem ter acontecido. O Turicas talvez tenha mais contexto.

**Reply by jamesperes** [reply]
Eu sou muito a favor da proposta, acho que o fato de um estudante pagar um valor menor e poder participar de ações tomadas nas assembleia é muito importante. Fazendo uma inclusão maior de todos.
Entendo o ponto do @luzfcb sobre os custos da APyB e acredito que o valor arrecadado por estudantes não tenha um impacto tão grande, e teremos a oportunidade de trazer mais pessoas para as assembleias.
Poderíamos até criar campanhas para trazer mais estudantes.

**Reply by maribedran** [reply]
A questão da saúde financeira da APyB não deveria impactar o direito dos membros de participar das decisões. Se a questão é dinheiro que se busque outras formas de financiamento ou aumente o valor da anuidade. Excluír estudantes dos processos democráticos da instituição não vai trazer magicamente mais dinheiro.

**Comment by tiidadavena** [comment]
Link para a ata: https://hackmd.io/x1pI84_fT8mPiTGIsS1WLA

**Comment by tiidadavena** [comment]
Repensar o formato da assembleia dada a interação entre as pautas e com a grande maioria dos direcionamentos sendo issues no Github Discussions. O Fundo para Inclusão é um grande exemplo, assim como o Flask Conf entre outros.

O único grande ponto seria a votação da cidade-sede que acredito que o formato atual atende.

**Reply by tiidadavena** [reply]
Tópico resultante pode ser visto em: #60

**Comment by tiidadavena** [comment]
## Links das issues resultantes da Assembleia

Conforme descrito na ata, os itens não foram debatidos a tempo foram sugeridos criarem novos tópicos. 

(A cada novo tópico criado, favor editar este comentário e add o link <3)

#52 
#60 
#62 
#64
